schema/rdl_schema_0.1.json

```diff
@@ -151,17 +151,56 @@
             "event_set": {
               "type": "object",
               "required": [
+                "id",
                 "hazard_type",
                 "analysis_type"
               ],
               "properties": {
+                "id": {
+                  "title": "Event set identifier",
+                  "type": "string",
+                  "description": "The unique identifier for this event set.",
+                  "minLength": 1
+                },
                 "hazard_type": {
-                  "$ref": "#/$defs/common_hazard_type",
-                  "description": "Type of hazard modelled"
+                  "title": "Hazard type",
+                  "type": "string",
+                  "description": "The main type of hazard phenomena in the modelled scenario(s).",
+                  "codelist": "hazard_type.csv",
+                  "openCodelist": false
                 },
                 "analysis_type": {
-                  "$ref": "#/$defs/common_analysis_type",
-                  "description": "Type of analysis, probabilistic or deterministic"
+                  "title": "Analysis type",
+                  "type": "string",
+                  "description": "The type of analysis used by the hazard model: probabilistic (multiple potential events characterised by different intensity and occurrence frequency); deterministic (frequency-independent representation, such as mean, median or max); empirical (obtained from real event observations).",
+                  "codelist": "analysis_type.csv",
+                  "openCodelist": false
```

Comment:
Add `enum` and link to codelist reference. Don't describe code semantics in field descriptions. Instead, the code descriptions will need to be integrated into the code descriptions in `analysis_type.csv` added in https://github.com/GFDRR/rdl-standard/pull/124/. If it's difficult to write a single description that works for both vulnerability and hazard, we can create separate CSV files or use document the code's meaning in each context within its description, e.g. for 'deterministic':

> **Hazard:** A frequency-independent representation, such as mean, median or max.
> **Vulnerability:** Deterministic approaches analyse disaster scenarios (e.g., historical or worst-case events) to estimate their impact, but do not consider the full range of possible outcomes and do not quantify the likelihood of each impact.

```suggestion
                  "type": "string",
                  "description": "The type of analysis used by the hazard model, from the closed [analysis type codelist]https://rdl-standard.readthedocs.io/en/{{version}}/reference/codelists#analysis_type).",
                  "codelist": "analysis_type.csv",
                  "openCodelist": false,
                  "enum": [
                    "deterministic",
                    "empirical",
                    "probabilistic"
                  ]
```